cms/models.py

```diff
@@ -80,6 +113,42 @@ def save(self, *args, **kwargs):
         self.slug = slugify("bootcamp-{}".format(self.product.klass_key))
         super().save(*args, **kwargs)
 
+class LearningAreasPage(BootcampProductChildPage):
+    """
+    Learning area's page for learning benefits.
+    """
+
+    subpage_types = []
+
+    heading = models.CharField(
+        max_length=250,
+        blank=False,
+        help_text="Heading highlighting the learning areas generally.",
+    )
+    sub_heading = RichTextField(
+        null=True, blank=True, help_text="Sub heading for learning areas."
+    )
+
+    body = RichTextField(
+        null=True, blank=True, help_text="body section for learning areas."
+    )
+
+    image = models.ForeignKey(
+        Image,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Learning Area image size must be at least 150x50 pixels.",
+    )
```

Comment:
Learning areas section requires exactly three images and rich text sections for each image.

Reply:
good catch 👍